addons/account/models/chart_template.py

```diff
@@ -186,6 +188,18 @@ def _load(self, template_code, company, install_demo):
         self._load_translations(companies=company)
         self._post_load_data(template_code, company, template_data)
 
+        # Add translations for names of new accounts without xmlid.
+        # This is (usually) i.e. necessary for the following accounts
+        #     * (some) bank utility accounts (see function _setup_utility_bank_accounts)
+        #     * cash / bank journal accounts (created "by" account.journal)
+        #     * unaffected earnings accounts (created "by" res.company)
+        ids_of_accounts_with_xmlid = self.env['ir.model.data'].search([('model', '=', 'account.account')]).mapped('res_id')
+        new_accounts_without_xmlid = self.env['account.account'].search([
```

Comment:
Why only for `account.account`? I didn't test, but I'm pretty sure that the journals have the same issue for instance.
Same for reconciliation models.